### PR TITLE
doc: add link to basic groot implementation

### PR DESCRIPTION
**Pull request by sbinet**

### CODE REVIEW

**Review by masonproffitt — APPROVED** [review]
Thanks for adding this! I added a bit to the description. If this looks okay to you, I'll go ahead and merge it.

File: README.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ Language implementations
 |----------|--------|-----------|
 |[opendata-benchmarks](https://github.com/stwunsch/opendata-benchmarks)|RDataFrame||
 |[nail](https://github.com/arizzi/nail/tree/master/benchmarks)|NAIL (Natual Analysis Implementation Language)||
+|[groot](https://github.com/go-hep/examples/tree/master/groot/bench-opendata)|[Go](https://golang.org)|Part of the [Go-HEP](https://go-hep.org/) project, `groot` is a pure Go package that provides read access to ROOT files|
```
Comment (sbinet) [inline README.md]: `s,read access,read/write access,`
And then LGTM :)

Thanks

Comment (masonproffitt) [reply README.md]: So it provides write access as well? I just pulled the line from https://godoc.org/go-hep.org/x/hep/groot. I guess the documentation needs to be updated.

Comment (sbinet) [reply README.md]: Yep. (It can not write ttrees yet. But soon...)

**Pull request merged**